<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="1" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Destino x:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Origen x:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Origen y:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" colspan="2">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Destino y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6" colspan="2">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Velocidad:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="2">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>RED:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>BLUE:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>GREEN:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
       <item row="6" column="7">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item row="4" column="7">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Ordenar Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Ordenar Distancia</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusds">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir_2"/>
   </widget>
   <widget class="QMenu" name="menuParticula">
    <property name="title">
     <string>Particula</string>
    </property>
    <addaction name="actionMostar"/>
    <addaction name="actionPuntos_Cercanos"/>
   </widget>
   <addaction name="menusds"/>
   <addaction name="menuParticula"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir_2">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionMostar">
   <property name="text">
    <string>Mostar</string>
   </property>
  </action>
  <action name="actionPuntos_Cercanos">
   <property name="text">
    <string>Puntos Cercanos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
